designsafe/apps/box_integration/integrations.py

```diff
@@ -3,9 +3,9 @@
 
 def provide_integrations():
     return [
-        {
-            'label': 'Box.com',
-            'href': reverse('box_integration:index'),
-            'description': 'Access files from your Box.com account in DesignSafe.',
-        },
```

Comment:
Instead of commenting out these lines in these `integrations` files, you should be able to just remove the app from `INSTALLED_APPS` in `common_settings.py`

Reply:
I tried that, it thrown errors like this one:
`RuntimeError: Model class designsafe.apps.box_integration.models.BoxUserToken doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.`

I could fix those errors but it implies to modify or delete some other parts of the code. I decided to take this approach since this is just a UI change I rather don't mess with functionality.